auth-helper/src/jwt.rs

```diff
@@ -104,58 +157,82 @@ impl Claims {
     }
 }
 
-/// Builder for the [`Claims`] structure.
-pub struct ClaimsBuilder {
-    iss: String,
-    sub: String,
-    aud: String,
-    exp: Option<u64>,
+pub trait BuildValidation: _sealed_validation::SealedBuildValidation {
```

Comment:
Can we put a `#[must_use]` here?

Reply:
This doesn't create a builder type or anything that must be used, so no?